### PR TITLE
Implemented Credibility

### PR DESCRIPTION
**Pull request by KanadeFuzen**
## Description

Basically made get_credibility functional. Seems left out even though it is in a notebook. Minh also said this would help increase the accuracy so here you go! Hope it works.

## Fixes

Fixed unfunctional get_credibility function.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: app/ocr.py [hunk 1]
```diff
@@ -457,11 +457,60 @@ def get_matches(input_list, topic, full_text):
...
+        if similar_narrow:
+            return 'Respondent was found credible'
+
+        elif similar_medium and similar_wide:
+            return 'Respondent was found credible'
+
+        else:
+            return 'Respondent was not found credible'
 
     def check_for_one_year(self) -> bool:
```
Comment (andrewlee977) [inline app/ocr.py:515]: Can this be deleted?

**Review by andrewlee977 — APPROVED** [review]
Looks good! Thanks for doing this.

File: app/ocr.py (hunk 1 above)
Comment (minh14496) [reply app/ocr.py:515]: Which part Andrew?

Comment (KanadeFuzen) [reply app/ocr.py:515]: Minh said to change it to True and False and I did. Is that more appropriate?

**Review by minh14496 — APPROVED** [review]
Perfect Waqas

**Pull request merged**